src/core/objective.jl

```diff
@@ -52,6 +52,46 @@ function objective_mc_min_load_setpoint_delta(pm::_PM.AbstractPowerModel)
 end
 
 
+"simplified minimum load delta objective (continuous load shed) with storage"
+function objective_mc_min_load_setpoint_delta_simple(pm::_PM.AbstractPowerModel)
+    for (n, nw_ref) in nws(pm)
+        var(pm, n)[:delta_pg] = Dict(i => JuMP.@variable(pm.model,
+                [c in conductor_ids(pm, n)], base_name="$(n)_$(i)_delta_pg",
+                start = 0.0) for i in ids(pm, n, :gen))
+
+        var(pm, n)[:delta_ps] = Dict(i => JuMP.@variable(pm.model,
+                [c in conductor_ids(pm, n)], base_name="$(n)_$(i)_delta_ps",
+                start = 0.0) for i in ids(pm, n, :storage))
+
+        for c in conductor_ids(pm, n)
+            for (i, gen) in nw_ref[:gen]
+                JuMP.@constraint(pm.model, var(pm, n, :delta_pg, i)[c] >=  (gen["pg"][c] - var(pm, n, :pg, i)[c]))
+                JuMP.@constraint(pm.model, var(pm, n, :delta_pg, i)[c] >= -(gen["pg"][c] - var(pm, n, :pg, i)[c]))
+            end
+
+            for (i, strg) in nw_ref[:storage]
+                JuMP.@constraint(pm.model, var(pm, n, :delta_ps, i)[c] >=  (strg["ps"][c] - var(pm, n, :ps, i)[c]))
+                JuMP.@constraint(pm.model, var(pm, n, :delta_ps, i)[c] >= -(strg["ps"][c] - var(pm, n, :ps, i)[c]))
+            end
+
+        end
+    end
+
+    load_weight = Dict(n => Dict(i => get(load, "weight", 1.0) for (i,load) in ref(pm, n, :load)) for n in nw_ids(pm))
+    M = Dict(n => Dict(c => 10*maximum([load_weight[n][i]*abs(load["pd"][c]) for (i,load) in ref(pm, n, :load)]) for c in conductor_ids(pm, n)) for n in nw_ids(pm))
+
+    JuMP.@objective(pm.model, Min,
+        sum(
+            sum(
+                sum( (            M[n][c]*(1 - var(pm, n, :z_demand, i))) for i in keys(nw_ref[:load])) +
+                sum( (abs(shunt["gs"][c])*(1 - var(pm, n, :z_shunt, i))) for (i,shunt) in nw_ref[:shunt]) +
+                sum( (                     var(pm, n, :delta_pg, i)[c] for i in keys(nw_ref[:gen]))) +
+                sum( (                     var(pm, n, :delta_ps, i)[c] for i in keys(nw_ref[:storage])))
```

Comment:
As a first pass on this objective I would drop these low priority penalty terms.  We can also consider dropping the `M` value on loads, we can consider the value of loads and shunts as equivalent.  The load weight priority could be put into the objective directly, or dropped entirely.

Reply:
To be clear, are you proposing the following?
```julia
    JuMP.@objective(pm.model, Min,
        sum(
            sum(
                sum( (1 - var(pm, n, :z_demand, i)) for i in keys(nw_ref[:load])) +
                sum( (1 - var(pm, n, :z_shunt, i)) for (i,shunt) in nw_ref[:shunt]) +
                sum(      var(pm, n, :delta_pg, i)[c] for i in keys(nw_ref[:gen])) +
                sum(      var(pm, n, :delta_ps, i)[c] for i in keys(nw_ref[:storage]))
            for c in conductor_ids(pm, n))
        for (n, nw_ref) in nws(pm))
    )
```

Reply:
Or are you suggesting that `:delta_pg` and `:delta_ps` also be dropped?

Reply:
How's this? (This is the most recent implementation.)

```julia
function objective_mc_min_load_setpoint_delta_simple(pm::_PM.AbstractPowerModel)
    JuMP.@objective(pm.model, Min,
        sum(
            sum(
                sum( ((1 - var(pm, n, :z_demand, i))) for i in keys(nw_ref[:load])) +
                sum( ((1 - var(pm, n, :z_shunt, i))) for (i,shunt) in nw_ref[:shunt])
            for c in conductor_ids(pm, n))
        for (n, nw_ref) in nws(pm))
    )
end
```